core/services/relay/evm/mercury/config_poller.go

```diff
@@ -87,10 +90,11 @@ type ConfigPoller struct {
 	filterName         string
 	destChainLogPoller logpoller.LogPoller
 	addr               common.Address
+	feedId             *common.Hash
 }
 
 // NewConfigPoller creates a new Mercury ConfigPoller
-func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address) (*ConfigPoller, error) {
+func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address, feedId *common.Hash) (*ConfigPoller, error) {
```

Comment:
Perhaps safer to make it value type, not pointer?

```suggestion
func NewConfigPoller(lggr logger.Logger, destChainPoller logpoller.LogPoller, addr common.Address, feedId common.Hash) (*ConfigPoller, error) {
```

Reply:
Agreed